website/docs/terms/ddl.md

```diff
@@ -0,0 +1,126 @@
+```
+
+In this example, you have to rename the `last_name` column [in jaffle_shop’s](https://github.com/dbt-labs/jaffle_shop) `customers` table to be called `last_initial`.
+
+### DROP
+
+The `DROP` command. Probably the most high-stakes DDL statement one can execute. One that should be used with the *utmost* of care. At its core, an executed `DROP` statement will remove that object from the data warehouse. You can DROP tables, views, schemas, databases, users, functions, and more.
+
+Some data warehouses such as Snowflake allow you to add restrictions to DROP statements to caution you about the impact of dropping a table, view, or schema before it’s actually dropped. In practice, we recommend you never drop raw source tables as they are often your baseline of truth. Your database user also usually needs the correct permissions to drop database objects.
+ 
+The syntax to use the DROP command is as follows:
+
+```sql
+DROP <database object type> <database object name>;
+```
+
+You can drop your `customer` table like this:
+
+```sql
+DROP TABLE customers;
+```
+
+### CREATE
+
+With the `CREATE` statement, you can create new objects in your data warehouse. The most common objects created with this statement are tables, schemas, views, and functions. Unlike `DROP`, `ALTER`, and `TRUNCATE` commands, there’s little risk with running `CREATE` statements since you can always drop what you create. 
+
+Creating tables and views with the `CREATE` command requires a strong understanding of how you want the data structured, including column name and data type. Using the `CREATE` command to establish tables and views can be laborious and repetitive, especially if the schema objects contain many columns, but is an effective way to create new objects in a database. After you create a table, you can use DML `INSERT` statements and/or a transformation tool such as dbt to actually get data in it.
+
+The generic syntax to use the `CREATE` command is as follows:
+
+```sql
+CREATE <database object type> <database object name>;
+```
+
+Creating a table using the CREATE statement may look a something like this:
+
+```sql
+CREATE TABLE prod.jaffle_shop.jaffles (
+	id varchar(255),
+	jaffle_name varchar(255)
+	created_at timestamp,
+	ingredients_list varchar(255),
+	is_active boolean
+);
+```
+
+Note that you had to explicitly define column names and column data type here. *You must have a strong understanding of your data’s structure when using the CREATE command for tables and views.*
+
+### TRUNCATE
+
+The `TRUNCATE` command will remove all rows from a table while maintaining the underlying table structure. The `TRUNCATE` command is only applicable for table objects in a database. Unlike `DROP` statements, `TRUNCATE` statements don’t remove the actual table from the database. While some platforms allow for `TRUNCATE` statements to be rolled-back, they do remove the actual data in tables and should be used with caution.
```

Comment:
This intro paragraph is from the old version. The new one should be: 

The TRUNCATE command will remove all rows from a table while maintaining the underlying table structure. The TRUNCATE command is only applicable for table objects in a database. Unlike DROP statements, TRUNCATE statements don’t remove the actual table from the database, just the data stored in them.

Reply:
Thank you!